src/vs/workbench/api/common/extHostSearch.ts

```diff
@@ -37,8 +38,27 @@ export class ExtHostSearch implements ExtHostSearchShape {
 	constructor(
 		@IExtHostRpcService private extHostRpc: IExtHostRpcService,
 		@IURITransformerService protected _uriTransformer: IURITransformerService,
-		@ILogService protected _logService: ILogService
-	) { }
+		@ILogService protected _logService: ILogService,
+		commands: ExtHostCommands,
+	) {
+		commands.registerArgumentProcessor({
+			processArgument: arg => {
+				if (arg && Array.isArray(arg)) {
+					return arg.map(matchContext => {
+						if (matchContext.$mid === 13 /* SearchViewContextMid */) {
```

Comment:
Could the 13 (magic number) be moved into a named constant to reveal its meaning